influunt-api/app/controllers/ControladoresController.java

```diff
@@ -311,6 +303,30 @@
 
     @Transactional
     @Dynamic(value = "ControladorAreaAuth(path)")
+    public CompletionStage<Result> finalizar(String id) {
+        Controlador controlador = Controlador.find.byId(UUID.fromString(id));
+        if (controlador == null) {
+            return CompletableFuture.completedFuture(notFound());
+        } else {
+            if(request().body().asJson() != null) {
```

Comment:
Se a descrição é obrigatória para finalizar o controlador, esse `if` deveria ser o contrário:
```
if (request().body().asJson() == null) {
    return CompletableFuture.completedFuture(status(UNPROCESSABLE_ENTITY,
                            Json.toJson(Collections.singletonList(new Erro("controlador", "Informe uma descrição para finalizar a configuração", "")))));
}
```

Pode aproveitar e tirar o `else` do `if (controlador == null)`. Se o controlador for null ele já vai retornar 404 e não vai executar o resto.